src/client/java/hadoop-daos/src/test/resources/run-java-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/sh
+
+#-Dthreads for running test concurrently in single JVM
+#-Djvms for running test in multiple JVMs
+
+pdsh -R ssh -w gsr[135-136] "/root/daos/java-test.sh read " \
```

Comment:
(lint) Ranges can only match single chars (mentioned due to duplicates). [SC2102]